util/dvsim/SimCfg.py

```diff
@@ -490,11 +503,12 @@ def gen_results_sub(items, results, fail_msgs):
     def gen_results_summary(self):
 
         # sim summary result has 4 columns provided by Testplan::results_table
-        table = [["Name", "Passing", "Total", "Pass Rate", "Detail"]]
+        table = [["Name", "Passing", "Total", "Pass Rate", "Coverage", "Detail"]]
```

Comment:
Perhaps we can remove the Detail column and use "name" column to link the results?